### PR TITLE
Updated the UIs of gadgets of sample 0 and sample 1

### PR DESCRIPTION
**Pull request by NipunaMarcus**

### CODE REVIEW

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js [hunk 1]
```diff
@@ -390,9 +393,19 @@ var initBarChart;
             rectangles = barGroup.selectAll(".bar").data(dataToProcess);
             rectangles.enter().append("rect")
                 .attr("class", "bar")
-                .attr("width", xScale.rangeBand())
+                .attr("width", function(){
+                    if(dataToProcess.length == 2){
+                        return xScale.rangeBand()/4;
```
Comment (sinthuja) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Formatting issue. There should be for '/'.

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js [hunk 2]
```diff
@@ -390,9 +393,19 @@ var initBarChart;
...
+                    }else {
+                        return xScale.rangeBand();
+                    }
+                })
                 .attr("x", function (d) {
-                    return xScale(d.name);
+                    if(dataToProcess.length == 2) {
+                        return xScale(d.name)+xScale.rangeBand()/2.5;
```
Comment (sinthuja) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Formatting issue, there should be a space before and after '+' and '/'.

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js [hunk 3]
```diff
@@ -424,7 +437,23 @@ var initBarChart;
                     d3.select(this)
                         .transition()
                         .duration(250)
-                        .attr("width", (xScale.rangeBand() + 3))
+                        .attr("fill", function (d) {
+                            return "rgb(107, 153, 185)";
+                        })
+                        .attr("width", function(){
+                            if(dataToProcess.length == 2){
+                                return xScale.rangeBand()/4;
```
Comment (sinthuja) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Formatting issue. There needs to be a space for '/'.

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js [hunk 4]
```diff
@@ -424,7 +437,23 @@ var initBarChart;
...
+                            }else {
+                                return xScale.rangeBand();
+                            }
+                        })
+                        .attr("x", function (d) {
+                            if(dataToProcess.length == 2) {
+                                return xScale(d.name)+xScale.rangeBand()/2.5;
```
Comment (sinthuja) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Formatting issue, there should be a space before and after '+' and '/'.

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js [hunk 5]
```diff
@@ -436,7 +465,23 @@ var initBarChart;
                     d3.select(this)
                         .transition()
                         .duration(250)
-                        .attr("width", xScale.rangeBand())
+                        .attr("fill", function (d) {
+                            return "rgb(31, 119, 180)";
+                        })
+                        .attr("width", function(){
+                            if(dataToProcess.length == 2){
+                                return xScale.rangeBand()/4;
```
Comment (sinthuja) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Same formatting issue mentioned up.

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js [hunk 6]
```diff
@@ -436,7 +465,23 @@ var initBarChart;
...
+                            }else {
+                                return xScale.rangeBand();
+                            }
+                        })
+                        .attr("x", function (d) {
+                            if(dataToProcess.length == 2) {
+                                return xScale(d.name)+xScale.rangeBand()/2.5;
```
Comment (sinthuja) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Same formatting issue mentioned up.

File: modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js [hunk 7]
```diff
@@ -425,7 +437,7 @@ var initDonutChart;
         var updateChartDetails = function (data) {
             $("#donutDetails").empty();
             data.forEach(function (element, index) {
-                var html = "<li>" + "<div style='background-color:" + color(index) + ";'></div><span>" + element.name + " - " + element.currentPopulation + "</span></li>";
+                var html = "<li>" + "<div style='background-color:" + color(element) + ";'></div><span>" + element.name + " - " + element.currentPopulation + "</span></li>";
```
Comment (sinthuja) [inline modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js]: This line seems very long, not readable. Break into two lines.

File: modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js [hunk 8]
```diff
@@ -507,7 +520,8 @@ var initDonutChart;
                 .attr("stroke", "white")
                 .attr("stroke-width", 0.5)
                 .attr("fill", function (d, i) {
-                    return color(i);
+                    console.log(d);
```
Comment (sinthuja) [inline modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js]: Having console.log() is not recommended, you can use it in the debugging but not in the final code. Please remove this..

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js (hunk 1 above)
Comment (NipunaMarcus) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js (hunk 2 above)
Comment (NipunaMarcus) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js (hunk 3 above)
Comment (NipunaMarcus) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js (hunk 4 above)
Comment (NipunaMarcus) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js (hunk 5 above)
Comment (NipunaMarcus) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js (hunk 6 above)
Comment (NipunaMarcus) [inline modules/samples/s0/gadgets/usa-population-by-age-and-gender/js/custom/bar-chart-gadget.js]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js (hunk 7 above)
Comment (NipunaMarcus) [inline modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

File: modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js (hunk 8 above)
Comment (NipunaMarcus) [inline modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js]: Fixed in the PR https://github.com/wso2/product-ds/pull/82

**Pull request merged**